client/lxd_operations.go

```diff
@@ -11,10 +11,15 @@ import (
 
 // GetOperationUUIDs returns a list of operation uuids.
 func (r *ProtocolLXD) GetOperationUUIDs() ([]string, error) {
+	err := r.CheckExtension("operations_get_query_all_projects")
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
This check needs to be removed as the function existed before introducing this new extension, and should continue working.

Reply:
so, is the extension still needed if this check should be removed here?

Reply:
Yes, the extension is still needed. As @roosterfish already said, it adds extra functionality and therefore we need one.